src/main/scala/is/hail/variant/Genotype.scala

```diff
@@ -584,27 +584,31 @@ object Genotype {
     Array(l0, l1, l2)
   }
 
-  lazy val phredConversionTable: Array[Double] = (0 to 65535).map { i => -10 * math.log10(if (i == 0) .25 else i) }.toArray
+  val dosageNorm: Double = 1 / 32768.0
 
-  lazy val dosageNorm: Double = 1 / 32768.0
+  val maxPhredInTable = 8192
+
+  lazy val linearToPhredConversionTable: Array[Double] = (0 to 65535).map { i => -10 * math.log10(if (i == 0) .25 else i) }.toArray
```

Comment:
No, linearToPhred is the other direction.  linear values can be between 0 and 65535.  phred values are not capped, so I'm using 8192 to make it a lookup in 99.99% of cases I've seen.